.gitignore

```diff
@@ -3,6 +3,7 @@
 *.so
 .DS_Store
 .f2py_f2cmap
+.idea/
```

Comment:
Why is this needed?

Reply:
Because I use PyCharm for its debugger, and PyCharm automatically creates the .idea/ folder in my workspace. I thought I'd add it there in case someone else did the same, and to avoid having to manually not commit the folder in my workspace.